.circleci/config.yml

```diff
@@ -0,0 +1,88 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+
+defaults: &defaults
+  working_directory: ~/rivet-collapsible
+  docker:
+  - image: circleci/node:7.10
+
+
+
+jobs:
+  build:
+    <<: *defaults
+
+    steps:
+    - checkout
+
+    # Download and cache dependencies
+    - restore_cache:
+        keys:
+        - v1-dependencies-{{ checksum "package.json" }}
+        # fallback to using the latest cache if no exact match is found
+        - v1-dependencies-
+
+    - run: npm install
+
+    - save_cache:
+        paths:
+        - node_modules
+        key: v1-dependencies-{{ checksum "package.json" }}
+
+    # run tests!
+    - run: npm run build
+
+  test:
+    <<: *defaults
+
+    steps:
+    - checkout
+
+    # Download and cache dependencies
+    - restore_cache:
+        keys:
+        - v1-dependencies-{{ checksum "package.json" }}
+        # fallback to using the latest cache if no exact match is found
+        - v1-dependencies-
+
+    - run: npm run build
+
+    - run:
+        command: node_modules/gulp/bin/gulp.js
+        background: true
+
+    # Run tests
+    - run:
```

Comment:
Whenever you run the tests locally you have to make sure the dev server is running. Will we need to do that somewhere here in order to run the tests as well or will it just automatically serve the files?